src/components/work/Footer/Footer.tsx

```diff
@@ -0,0 +1,97 @@
+          </svg>
+        </Col>
+
+        {contentData.map(item => (
+          <Col sm="3" lg="2" className="mb-4 mb-lg-0" key={item.header}>
+            <ListGroup className="list-group-borderless" flush>
+              <ListGroupItemHeading className="h6 font-weight-semi-bold">{item.header}</ListGroupItemHeading>
+              {item.links.map(link => (
+                <ListGroupItem action tag={Link} to={link[1]} key={link[0]}>
+                  {link[0]}
+                </ListGroupItem>
+              ))}
+            </ListGroup>
+          </Col>
+        ))}
+
+        <Col sm="3" lg="3" className="mb-4 mb-lg-0">
+          <ListGroup className="list-group-borderless" flush>
+            <ListGroupItemHeading className="h6 font-weight-semi-bold">Contact</ListGroupItemHeading>
+            <ListGroupItem className="text-secondary">+1 (408) 123-4567</ListGroupItem>
+            <ListGroupItem tag={Link} to="/work">
+              support@beenest.com
+            </ListGroupItem>
+            <ListGroupItem className="text-secondary">
+              717 Market St.
+              <br />
+              San Francisco, CA 94103
+            </ListGroupItem>
+          </ListGroup>
+        </Col>
+      </Row>
+    </Container>
+
+    <Container className="space-1 d-flex flex-column flex-md-row justify-content-sm-between align-items-center align-items-md-end">
+      <div className="d-flex d-md-none mb-3">
+        <svg width="190px" height="36px" viewBox="0 0 190 36" fill="#377dff">
```

Comment:
what is this SVG? can it be a separate component?

Reply:
👍